tools/joffreBulkBooking.js

```diff
@@ -0,0 +1,139 @@
+const AWS = require('aws-sdk');
+const readline = require('readline');
+
+const TABLE_NAME = process.env.TABLE_NAME || 'parksreso';
+
+const options = {
+  region: 'ca-central-1',
+  // endpoint: 'http://localhost:8000'
+};
+
+// todo: need to confirm correct values for parkName, facilityName, email, firstName, lastName
+const parkName = 'Joffre Lakes Provincial Park';
+const facilityName = 'Day-Use Trails';
+const email = 'parkinfo@gov.bc.ca';
```

Comment:
Nit - Do we need this to be `parkinfo@gov.bc.ca`? Bulk reminder email changes coming in the near future will be querying the DB for reserved passes and sending reminders to the emails associated with these passes. This should probably not be an actual email address.

Reply:
The email was changed to noreply@gov.bc.ca